values.yaml

```diff
@@ -15,7 +15,7 @@ docker:
 ##
 cog:
   name: cog
-  serviceType: ClusterIP
+  serviceType: NodePort
```

Comment:
`NodePort` maps an external port to each node in your cluster, and is really only necessary if you want to directly access a given service from outside. `ClusterIP` is an internal virtual IP for accessing the service.

I'd argue that external access in this scenario will always go through the nginx service making `NodePort` unnecessary.